oas

```diff
@@ -696,7 +693,7 @@
             "type": "string",
             "description": "end-user-friendly message describing the status. In the special case where status is CONFIGURATION_REQUESTED, the message MUST be a browser URL to a configuration UI where further configuration can take place"
           },
-          "recommendations": {
+          "recommendation": {
```

Comment:
Why "recommendation" (without s) when it is an array of Recommendation objects. Also, the example show "recommendations"

Reply:
As we have structured the API it will result in 1 recommendation regarding the next irrigation event. To keep consistency with other advisors I will keep the (s) but in truth in the case of irrigation_advisor "recommendations" is not an array but a single ibject.